core/store/models/log_events_test.go

```diff
@@ -393,30 +393,6 @@ func TestFilterQueryFactory_InitiatorRunLog(t *testing.T) {
 	assert.Equal(t, want, filter)
 }
 
-func TestFilterQueryFactory_InitiatorFluxMonitor(t *testing.T) {
```

Comment:
(Same as above) Per a conversation with @se3000, We no longer use `FilterQueryFactory` to handle FluxAggregator log queries.